liblwgeom/lwgeom_geos_cluster.c

```diff
@@ -346,10 +349,16 @@ union_dbscan_minpoints_1(LWGEOM** geoms, uint32_t num_geoms, UNIONFIND* uf, doub
 
 	for (p = 0; p < num_geoms; p++)
 	{
+		int rv = LW_SUCCESS;
 		if (lwgeom_is_empty(geoms[p]))
 			continue;
 
-		dbscan_update_context(tree.tree, &cxt, geoms, p, eps);
+		rv = dbscan_update_context(tree.tree, &cxt, geoms, p, eps);
```

Comment:
Smaller than a nit, but...can't the variable be defined here in C99? (`int rv = dbascan_update_context...`)

Reply:
I think it can, but I think we also have some bots building with older compilers and generating errors/warnings for doing C99 stuff like that. The discipline of the bots!